apps/src/block_utils.js

```diff
@@ -1074,16 +1075,17 @@ exports.createJsWrapperBlockCreator = function (
       helpUrl: getHelpUrl(docFunc), // optional param
       init: function () {
         // Styles should be used over hard-coded colors in Google Blockly blocks
-        if (style && this.setStyle) {
-          this.setStyle(style);
-        } else if (color) {
+        if (color) {
           Blockly.cdoUtils.setHSV(this, ...color);
         } else if (!returnType) {
-          if (this.setStyle) {
-            this.setStyle('default');
-          } else {
-            Blockly.cdoUtils.setHSV(this, ...DEFAULT_COLOR);
-          }
+          // CDO Blockly automatically sets block color for blocks with a specific
+          // returnType (e.g. yellow for "Location")
+          Blockly.cdoUtils.setHSV(this, ...DEFAULT_COLOR);
+        }
+        if (style) {
```

Comment:
nit: The syntax `this.setStyle(style ? style : 'default');` should also work.

Reply:
Or `this.setStyle(style || 'default');`?

Reply:
I get nervous about using the `||` operator like that for some reason, but I think it's unfounded. I think it's because I know the `?` operator forces the thing before it to be interpreted as either truthy or falsey, so it feels more readable to me in terms of explicitly showing the intent. I have more trouble remembering how JS treats falsy values with logical operators. But, I think you're right that they're the same here.

<img width="287" alt="Screenshot 2024-01-26 at 5 11 29 PM" src="https://github.com/code-dot-org/code-dot-org/assets/26844240/d8f7e2bf-f1c2-4e59-a985-da5ca4be70e5">

Now that I think about this a little more, I think I'm wary of `&&` and `||` because I'm overreacting to well-known bug in React where `0 && something` doesn't do what you want. Here's a link, just in case you're curious: https://www.codemzy.com/blog/react-render-0-conditional